src/Stripe.net/Entities/Events/Event.cs

```diff
@@ -4,36 +4,73 @@ namespace Stripe
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
+    [JsonConverter(typeof(EventConverter))]
     public class Event : StripeEntity, IHasId, IHasObject
     {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// String representing the object’s type. Objects of the same type share the same value.
+        /// </summary>
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        /// <summary>
+        /// For Connect events, the identifier of the account that emitted the event.
+        /// </summary>
         [JsonProperty("account")]
         public string Account { get; set; }
 
+        /// <summary>
+        /// The Stripe API version used to render <c>data</c>.
+        /// </summary>
+        /// <remarks>
+        /// Note: This property is populated only for events on or after October 31, 2014.
+        /// </remarks>
         [JsonProperty("api_version")]
         public string ApiVersion { get; set; }
 
+        /// <summary>
+        /// Time at which the object was created.
+        /// </summary>
         [JsonProperty("created")]
         [JsonConverter(typeof(DateTimeConverter))]
         public DateTime? Created { get; set; }
 
+        /// <summary>
+        /// Object containing data associated with the event.
+        /// </summary>
         [JsonProperty("data")]
         public EventData Data { get; set; }
 
+        /// <summary>
+        /// Has the value <c>true</c> if the object exists in live mode or the value <c>false</c> if
+        /// the object exists in test mode.
+        /// </summary>
         [JsonProperty("livemode")]
         public bool Livemode { get; set; }
 
+        /// <summary>
+        /// Number of webhooks that have yet to be successfully delivered (i.e., to return a 20x
```

Comment:
Ugh I don't like this wording. I know it's the official API docs one but we really need to automate the comments generation at least at some point in the future.

Reply:
+1000. I started looking into this, it's not a trivial problem but it's definitely doable.